cpp/src/io/comp/gpuinflate.cu

```diff
@@ -45,72 +45,76 @@ Mark Adler    madler@alumni.caltech.edu
 
 #include <io/utilities/block_utils.cuh>
 #include "gpuinflate.h"
+#include "io_uncomp.h"
 
 namespace cudf {
 namespace io {
-#define NUMTHREADS 128  // Threads per block
 
-#define MAXBITS 15                        // maximum bits in a code
-#define MAXLCODES 286                     // maximum number of literal/length codes
-#define MAXDCODES 30                      // maximum number of distance codes
-#define MAXCODES (MAXLCODES + MAXDCODES)  // maximum codes lengths to read
-#define FIXLCODES 288                     // number of fixed literal/length codes
+constexpr int maxbits   = 15;   // maximum bits in a code
+constexpr int maxlcodes = 286;  // maximum number of literal/length codes
+constexpr int maxdcodes = 30;   // maximum number of distance codes
+constexpr int fixlcodes = 288;  // number of fixed literal/length codes
```

Comment:
Ooh, comments like this would help in all the other places we have these magic constexprs...

```suggestion
constexpr int max_bits   = 15;   // maximum bits in a code
constexpr int max_l_codes = 286;  // maximum number of literal/length codes
constexpr int max_d_codes = 30;   // maximum number of distance codes
constexpr int fix_l_codes = 288;  // number of fixed literal/length codes
```